### PR TITLE
Add new troubleshooting: removing finalizers

### PR DESCRIPTION
**Pull request by superojla**
**Description**

Changes proposed in this pull request:

- Add troubleshooting guide for removing the finalizers

**Related issue(s)**

#6840

### CODE REVIEW

File: docs/kyma/10-02-basic-troubleshooting.md [hunk 1]
```diff
@@ -54,3 +54,18 @@ minikube ssh -- docker run -i --rm --privileged --pid=host debian nsenter -t 1 -
 
 If you restart Kyma using unauthorized methods, such as triggering the installation when a Minikube cluster with Kyma is already running, the cluster might become unresponsive which can be fixed by reinstalling Kyma.
 To prevent such behavior, [stop and restart Kyma](#installation-install-kyma-locally-stop-and-restart-kyma-without-reinstalling) using only the method described.
+
+## Unable to deprovision Gardener cluster
+
+If you are unable to deprovision a Gardener cluster and you receive the following error:
```
Comment (majakurcius) [inline docs/kyma/10-02-basic-troubleshooting.md:60]:
```suggestion
If you can't deprovision a Gardener cluster, you might receive the following error:
```
- `can't`: I know I suggested `unable` in the first place, but you were right, it's better to be consistent with the rest of the document.

- `and you receive the following error` -> `, you might receive the following error:`: as a general rule, we don't split sentences with code blocks.

File: docs/kyma/10-02-basic-troubleshooting.md [hunk 2]
```diff
@@ -54,3 +54,18 @@ minikube ssh -- docker run -i --rm --privileged --pid=host debian nsenter -t 1 -
...
+
+```
```
Comment (majakurcius) [inline docs/kyma/10-02-basic-troubleshooting.md:62]:
```suggestion
```bash
```
This improves rendering. It's not strictly necessary, but we generally try to include it whenever possible. 
You might want to add it to the rest of the code blocks here.

File: docs/kyma/10-02-basic-troubleshooting.md [hunk 3]
```diff
@@ -54,3 +54,18 @@ minikube ssh -- docker run -i --rm --privileged --pid=host debian nsenter -t 1 -
...
+Flow "Shoot cluster deletion" encountered task errors: [task "Cleaning extended API groups" failed: 1 error occurred:
+retry failed with context deadline exceeded, last error: remaining objects are still present: [*v1beta1.CustomResourceDefinition /installations.installer.kyma-project.io]
+```
+
+You need to first remove the finalizer from `kyma-installation` CR. Run this command:
```
Comment (majakurcius) [inline docs/kyma/10-02-basic-troubleshooting.md:67]:
```suggestion
If this happens, you must first remove the finalizer from `kyma-installation` CR. Run this command: 
```

File: docs/kyma/10-02-basic-troubleshooting.md [hunk 4]
```diff
@@ -54,3 +54,18 @@ minikube ssh -- docker run -i --rm --privileged --pid=host debian nsenter -t 1 -
 
 If you restart Kyma using unauthorized methods, such as triggering the installation when a Minikube cluster with Kyma is already running, the cluster might become unresponsive which can be fixed by reinstalling Kyma.
 To prevent such behavior, [stop and restart Kyma](#installation-install-kyma-locally-stop-and-restart-kyma-without-reinstalling) using only the method described.
+
+## Unable to deprovision Gardener cluster
```
Comment (majakurcius) [inline docs/kyma/10-02-basic-troubleshooting.md:58]:
```suggestion
## Can't deprovision Gardener cluster
```
I know I suggested `unable` in the first place, but you were right, it's better to be consistent with the rest of the document.

File: docs/kyma/10-02-basic-troubleshooting.md [hunk 5]
```diff
@@ -54,3 +54,18 @@ minikube ssh -- docker run -i --rm --privileged --pid=host debian nsenter -t 1 -
...
+If you are unable to deprovision a Gardener cluster, you might receive the following error:
+
+```bash
+Flow "Shoot cluster deletion" encountered task errors: [task "Cleaning extended API groups" failed: 1 error occurred:
+retry failed with context deadline exceeded, last error: remaining objects are still present: [*v1beta1.CustomResourceDefinition /installations.installer.kyma-project.io]
+```
+
+If this happens, you must first remove the finalizer from `kyma-installation` CR. Run this command: 
+
+```
```
Comment (superojla) [inline docs/kyma/10-02-basic-troubleshooting.md:69]: add ```bash

Comment (superojla) [inline docs/kyma/10-02-basic-troubleshooting.md:69]:
```suggestion
```bash
```

File: docs/kyma/10-02-basic-troubleshooting.md [hunk 6]
```diff
@@ -24,33 +24,52 @@ scripts/is-installed.sh --verbose
...
+## Can't deprovision Gardener cluster
+
+If you are unable to deprovision a Gardener cluster, you might receive the following error:
+
+```bash
+Flow "Shoot cluster deletion" encountered task errors: [task "Cleaning extended API groups" failed: 1 error occurred:
+retry failed with context deadline exceeded, last error: remaining objects are still present: [*v1beta1.CustomResourceDefinition /installations.installer.kyma-project.io]
+```
+
+If this happens, you must first remove the finalizer from `kyma-installation` CR. Run this command:
```
Comment (majakurcius) [inline docs/kyma/10-02-basic-troubleshooting.md:71]:
```suggestion
If this happens, you must first remove the finalizer from the `kyma-installation` CR. Run this command: 
```

**Review by majakurcius — APPROVED** [review]
🚀

**Review by jakkab — APPROVED** [review]
lgtm

File: docs/kyma/10-02-basic-troubleshooting.md [hunk 7]
```diff
@@ -24,33 +24,52 @@ scripts/is-installed.sh --verbose
...
+```bash
+Flow "Shoot cluster deletion" encountered task errors: [task "Cleaning extended API groups" failed: 1 error occurred:
+retry failed with context deadline exceeded, last error: remaining objects are still present: [*v1beta1.CustomResourceDefinition /installations.installer.kyma-project.io]
+```
+
+If this happens, you must first remove the finalizer from the `kyma-installation` CR. Run this command: 
+
+```bash
+kubectl patch installation kyma-installation --type=merge -p '{"metadata":{"finalizers":null}}'
+```
```
Comment (piotrmsc) [inline docs/kyma/10-02-basic-troubleshooting.md:75]: I first run this command what should I do next? What is the second step?

**Review by piotrmsc — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]
🚀

**Pull request merged**